src/js/express.js

```diff
@@ -135,7 +149,21 @@ fluid.defaults("gpii.express", {
     gradeNames: ["fluid.modelComponent", "gpii.express.expressConfigHolder"],
     members: {
         directChildrenOfInterest: [],
-        childrenByParent:         {}
+        childrenByParent:         {},
+        views: {
```

Comment:
Whilst we are in general in favour of declarative definitions, this is too much material to be usefully encoded this way. In general, when you find yourself writing two nested expanders, it is time to fall back to code!

views: "@expand:gpii.express.expandViews({that}.options.config.express.views)" 

with 
gpii.express.resolvePaths = function (views) {
   return fluid.transform(fluid.makeArray(views), fluid.module.resolvePath);
};

is more readable and more moral - with the benefit that you can now reuse the "resolvePaths" function in the other place where it is required in handlebars